client/my-sites/checkout/checkout/payment-chat-button.jsx

```diff
@@ -8,7 +8,7 @@ import Gridicon from 'gridicons';
 /**
  * Internal dependencies
  */
-import HappychatButton from 'components/happychat/button';
+import HappychatButton from 'extensions/happychat/components/button';
```

Comment:
Same here, if we aren't able to remove the direct references to the extension, we should at least comment prominently that it needs to be resolved.